be/src/service/backend_service.cpp

```diff
@@ -291,6 +291,15 @@ void BackendService::get_next(TScanBatchResult& result_, const TScanNextBatchPar
         TUniqueId fragment_instance_id = context->fragment_instance_id;
         std::shared_ptr<arrow::RecordBatch> record_batch;
         bool eos;
+        Status queue_status = _exec_env->result_queue_mgr()->queue_status(fragment_instance_id);
```

Comment:
I don't think it is necessary to call this function separately, fetch_result should be able to return status

Reply:
Done